lib/scenic/adapters/postgres.rb

```diff
@@ -46,6 +48,16 @@ def create_view(name, sql_definition)
         execute "CREATE VIEW #{name} AS #{sql_definition};"
       end
 
+      # Updates a view in the database.
```

Comment:
Not according to the docs

> The new query must generate the same columns that were generated by the existing view query (that is, the same column names in the same order and with the same data types), but it may add additional columns to the end of the list.